internal/cmdexport/cmdexport.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// package cmdexport contains the export command.
+package cmdexport
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// TODO: update
+const KptImage = "gongpu/kpt:latest"
+
+func ExportCommand() *cobra.Command {
+	return GetExportRunner().Command
+}
+
+func GetExportRunner() *ExportRunner {
+	r := &ExportRunner{}
+	c := &cobra.Command{
+		Use: "export orchestrator DIR/",
+		// TODO:
+		Short:   "",
+		Long:    "",
+		Example: "",
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 2 {
+				return fmt.Errorf("accepts %d args, received %d", 2, len(args))
+			}
+
+			r.Orchestrator, r.Dir = args[0], args[1]
+
+			switch r.Orchestrator {
+			case "github-actions":
```

Comment:
You can change this to a constant or to an enumerated type
```
const (
    GitHub-Actions = iota
    CloudBuild
    GitLabCI
    (others)
)
```

Reply:
Considering we mainly deal with string types here, using `enum` may not be so helpful. So I will keep it as it is.